### ISSUE TITLE
bug: "Layout: expected an options Object, but got: undefined"

### ISSUE THREAD

**Issue by wires** [kettanaito/atomic-layout#173]
## What:

After installing `atomic-layout`, I cannot use it.

```
$ npm install atomic-layout --save
$ node
> let {Component} = require('atomic-layout')
Failed to configure Layout: expected an options Object, but got: undefined.
```

## Why:

A call to `invariant` requires an options dictionary to be passed[[1]](https://github.com/kettanaito/atomic-layout/blob/master/src/Layout.ts#L32), but `Layout.ts` directly exports `new Layout()`[[2]](https://github.com/kettanaito/atomic-layout/blob/master/src/Layout.ts#L87), so this renders the library unusable.

## Environment:

- node version: `v11.7.0`
- npm version: `6.9.0`
- atomic-layout version: `0.8.0`

## Steps to reproduce:

See **What**.

## Expected behavior:

I expect to be able to pass arguments to Layout or at least have no-options-passed work :-)

**Comment by wires** [comment]
I created a PR because I wanted to try out the library, I'm not sure if you want to fix it this way because I don't know what the intended invariant should be.

**Comment by kettanaito** [comment]
Hi, @wires. Thank you for reporting this.

I've provided with some technical details in your pull request. The issue has been fixed, I plan to publish it in the next patch version today. Worry not, it's a wrong handling of Layout constructor that has no affect over the library's functionality.

**Comment by kettanaito** [comment]
Fix published under `0.8.1`. Please update to that version or newer. Thank you.
